Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -2202,7 +2138,7 @@ extension StatusTableViewController {
 extension StatusTableViewController: BluetoothObserver {
     func bluetoothDidUpdateState(_ state: BluetoothState) {
         refreshContext.update(with: .status)
-        reloadData(animated: true)
+        Task { await reloadData(animated: true) }
```

Comment:
This does run the reloadData method on MainActor, because UIViewControllers are already marked @MainActor. I may try to clean it up a bit, to avoid the await; maybe annotating the closure is better than awaiting.